src/components/Announce/index.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+import strings from 'lang';
+import { connect } from 'react-redux';
+import { getGithubPulls } from 'actions/githubPulls';
+import ReactMarkdown from 'react-markdown';
+import { IconGithub } from 'components/Icons';
+
+import styles from './styles.css';
+
+const Announce = ({ title, body, onClick, link }) => (
+  <div className={styles.announce}>
+    <main>
+      <h4><a href={link} title={strings.announce_github_more}><IconGithub /></a> {title}</h4>
+      {body && <ReactMarkdown source={body} />}
+    </main>
+    <aside>
+      <RaisedButton
+        backgroundColor={styles.blue}
+        onClick={onClick}
+        label={strings.announce_dismiss}
+      />
+    </aside>
+  </div>
+);
+
+class RequestLayer extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      isDismissed: false,
+    };
+
+    this.dismiss = (value) => {
+      if (localStorage) {
+        this.setState({
+          isDismissed: true,
+        });
+        localStorage.setItem('dismiss', value);
+      }
+    };
+
+    this.getDate = (days) => {
+      const msPerDay = 24 * 60 * 60 * 1000;
+
+      const date = new Date(new Date() - (msPerDay * days));
+
+      // Date formatting must follow the ISO8601 standard, which is YYYY-MM-DD--that's year-month-day
```

Comment:
maybe just use this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString